{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)"]}],"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Mac Catalyst"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"network","kind":"identifier"},{"kind":"text","text":"("},{"kind":"externalParam","text":"cause"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"CommunicatorError","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"text":")","kind":"text"}],"languages":["swift"]}]}],"sections":[],"schemaVersion":{"patch":0,"major":0,"minor":3},"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"network","kind":"identifier"},{"text":"(","kind":"text"},{"text":"cause","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO","text":"CommunicatorError"},{"text":")","kind":"text"}],"role":"symbol","roleHeading":"Case","title":"NfinityCloudError.network(cause:)","modules":[{"name":"PasswordAndNfinityFramework"}],"symbolKind":"case","externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO7networkyAC012CommunicatorD00hF0O_tcACmF"},"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)"},"abstract":[{"type":"text","text":"Case represents network error"}],"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/network(cause:)":{"title":"NfinityCloudError.network(cause:)","type":"topic","abstract":[{"text":"Case represents network error","type":"text"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"network"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"text":")","kind":"text"}],"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"role":"symbol","type":"topic","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"NfinityCloudError"}],"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","title":"NfinityCloudError","kind":"symbol","abstract":[{"type":"text","text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud."}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"abstract":[],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","type":"topic","url":"\/documentation\/passwordandnfinityframework","role":"collection","kind":"symbol","title":"PasswordAndNfinityFramework"}}}